go/exocom/main_test.go

```diff
@@ -0,0 +1,128 @@
+package main_test
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/DATA-DOG/godog"
+	"github.com/Originate/exocom/go/exocom/test_helpers"
+)
+
+// nolint gocyclo
+func FeatureContext(s *godog.Suite) {
+	var cmd *exec.Cmd
+	var cmdOutput string
+	var cmdStdout io.ReadCloser
+	var cmdStderr io.ReadCloser
```

Comment:
Thoughts on using a global `process` like [here](https://github.com/Originate/exosphere/blob/master/exo-go/test_helpers/shared_feature_context.go#L20) instead of using 4 global variables?